config.go

```diff
@@ -264,6 +269,15 @@ func loadConfig(path string) (*config, error) {
 	return c, nil
 }
 
+func firstNonEmptyInt(ints ...int) int {
```

Comment:
See above, think you need to use pointers.

If you dont, you should change the name of this to firstNonZeroInt